### PR TITLE
patch score in backend when edited in the frontend and i18n for page

### PR DESCRIPTION
**Pull request by Gerwoud**

### CODE REVIEW

File: frontend/src/components/ProjectSubmissionOverview/ProjectSubmissionOverviewDatagrid.tsx [hunk 1]
```diff
@@ -27,9 +27,28 @@ const fetchSubmissionsFromUser = async (submission_id: string) => {
     });
 };
 
+const editGrade = (submission: Submission, oldSubmission: Submission) => {
+  const submission_id = submission.submission_id;
+  const newGrade = submission.grading;
+
+  if (newGrade < 0 || newGrade > 20) {
+    return oldSubmission;
```
Comment (Vucis) [inline frontend/src/components/ProjectSubmissionOverview/ProjectSubmissionOverviewDatagrid.tsx:35]: Does this get caught anywhere so people know that the submission wasn't changed?

Comment (AronBuzogany) [reply frontend/src/components/ProjectSubmissionOverview/ProjectSubmissionOverviewDatagrid.tsx:35]: What he said

**Review by AronBuzogany — CHANGES_REQUESTED** [review]

File: frontend/src/components/ProjectSubmissionOverview/ProjectSubmissionOverviewDatagrid.tsx [hunk 2]
```diff
@@ -27,57 +29,88 @@ const fetchSubmissionsFromUser = async (submission_id: string) => {
...
-      </IconButton>
-    ),
-  },
-];
+const editGrade = (submission: Submission, oldSubmission: Submission, errorMessage: string) => {
+  const submission_id = submission.submission_id;
+  const newGrade = submission.grading;
+
+  if (newGrade < 0 || newGrade > 20) {
+    alert(errorMessage);
```
Comment (AronBuzogany) [inline frontend/src/components/ProjectSubmissionOverview/ProjectSubmissionOverviewDatagrid.tsx:37]: No using alert

**Review by AronBuzogany — CHANGES_REQUESTED** [review]

File: frontend/src/components/ProjectSubmissionOverview/ProjectSubmissionOverviewDatagrid.tsx (hunk 2 above)
Comment (Gerwoud) [reply frontend/src/components/ProjectSubmissionOverview/ProjectSubmissionOverviewDatagrid.tsx:37]: What do i need to use then?

Comment (Gerwoud) [reply frontend/src/components/ProjectSubmissionOverview/ProjectSubmissionOverviewDatagrid.tsx]: https://github.com/SELab-2/UGent-3/pull/355/commits/f74ea45daaf697b134a0b9ccc2390f81d0e529dd

**Review by AronBuzogany — APPROVED** [review]

**Pull request merged**